generalTable.Rmd

```diff
@@ -14,7 +14,7 @@ output: html_document
 
 The following table contains a list of general informatics and biomedical platforms that can help researchers work with publicly available data and or their own data. General platforms are those which work with a wide variety of data types, from imaging data to “omics” data, and provide computing resource support across different areas of bioinformatics research. 
```

Comment:
```suggestion
The following table contains a list of informatics and biomedical platforms that can help researchers work with **multiple types** of publicly available data and or their own data. These platforms provide computing resource support across different areas of bioinformatics research. 
```

Reply:
Same here as well. This is now called multiTable.Rmd